src/odemis/gui/model/__init__.py

```diff
@@ -1453,7 +1453,7 @@ def moveFocusRel(self, shift):
         # FIXME: "stop all axes" should also clear the queue
 
         # If streams have a z-level, we calculate the shift differently.
-        
+
```

Comment:
Somehow I cannot change these lines back to the original. Maybe someone added tabs here instead of 4 spaces?

Reply:
This means there were 8 empty spaces on that line and you removed those spaces. So now it's a completely empty line.

Reply:
I tried again to add 8 spaces instead of leaving the line empty and it still pop-up....

Reply:
Probably this is a pycharm setting which converts these things during committing, I will check those settings.

Reply:
Maybe other people disagree, but I don't think it's bad to do this. Especially since it's only on two lines in your PR, it doesn't mess up the entire PR. And it's cleaner to have completely empty lines, which you will now have. 

In Pycharm settings you can go to Editor > General, then all the way at the bottom there is a section On Save. I think if that is set to Remove trailing spaces on all lines, it will also remove spaces on empty lines.

Reply:
Indeed, please don't spend too much time on that. It's better to not fix whitespace in the same commit, but it's even worse to spend 1h to put back whitespace ;-)
Your PR is still readable, so let's not worry too much.

Reply:
>And it's cleaner to have completely empty lines, which you will now have. [From @tepals]
>but it's even worse to spend 1h to put back whitespace ;-) [ From @pieleric]
Great, I wasn't particularly looking forward to spending more time on trying to fix this.